test/unit/test_disco_storage.py

```diff
@@ -153,13 +152,35 @@ def test_take_snapshot(self):
             InstanceId=instance.instance_id,
             Device='/dev/sdb'
         )
+        return volume['VolumeId']
 
-        snapshot_id = self.storage.take_snapshot(volume_id=volume['VolumeId'])
-
+    def _test_take_snapshot_validate_results(self, snapshot_id, tags):
```

Comment:
Same comment as a above. Its not the actual unit test so probably remove `_test` from the name. How about `_validate_snapshot_fields`